documentation/docs/03_account.md

```diff
@@ -27,6 +27,41 @@ Prints the account balance.
 > Account "main": balance
 ```
 
+### `burn-native-token`
+
+Burns a native token.
+
+#### Parameters
+
+| Name       | Optional  | Default | Example    |
+| ---------- | --------- | ------- | ---------- |
+| `token_id` | ✘         | N/A     | "0xabcdef" |
```

Comment:
Nitpicking but this example is wrong, let's not give incorrect info, just reuse the one from below

Reply:
You started with that :carlos:

Reply:
I just did for foundry metadata but there it's a correct value